js/tests/unit/modal.js

```diff
@@ -167,7 +167,7 @@ $(function () {
         assert.notEqual($('#modal-test').length, 0, 'modal insterted into dom')
         $('.contents').click()
         assert.ok($('#modal-test').is(':visible'), 'modal visible')
-        $('#modal-test .modal-backdrop').click()
```

Comment:
Well, you'll never click on the backdrop. The event listener for the `click` event is on the modal element.